test/nexus_test.go

```diff
@@ -897,6 +897,83 @@ func TestWorkflowTestSuite_WorkflowRunOperation(t *testing.T) {
 	})
 }
 
+func TestWorkflowTestSuite_WorkflowRunOperation_ScheduleToCloseTimeout(t *testing.T) {
+	sleepDuration := 500 * time.Millisecond
+	handlerWF := func(ctx workflow.Context, _ nexus.NoValue) (nexus.NoValue, error) {
+		return nil, workflow.Sleep(ctx, sleepDuration)
+	}
+
+	op := temporalnexus.NewWorkflowRunOperation(
+		"op",
+		handlerWF,
+		func(ctx context.Context, _ nexus.NoValue, opts nexus.StartOperationOptions) (client.StartWorkflowOptions, error) {
+			return client.StartWorkflowOptions{ID: opts.RequestID}, nil
```

Comment:
Can you put a 100ms sleep here (don't have to commit it) just to verify that time is not skipped while this is running?

Reply:
That's a good call actually. I added another test to timeout before the operation starts.